onnxmltools/utils/tests_dl_helper.py

```diff
@@ -6,42 +6,69 @@
 """
 Helpers to help testing deep learning converted models.
 """
+import six
```

Comment:
Please swap `six` and `numpy as np`.

Reply:
Two different imports, and do you think it looks nice from short to the long.